src/bin/good-fences.rs

```diff
@@ -1,32 +1,106 @@
 extern crate good_fences_rs_core; // Optional in Rust 2018
 extern crate serde_json;
+use clap::{self, Parser};
+use good_fences_rs_core::evaluate_fences::ImportRuleViolation;
 use good_fences_rs_core::good_fences_runner::GoodFencesRunner;
-use std::env::set_current_dir;
+use serde::Serialize;
+use std::env::{self, set_current_dir};
```

Comment:
just `use std::env;` here and reference `set_current_dir` as `env::set_current_dir` for consistency when addressing symbols